src/python/pants/option/global_options.py

```diff
@@ -1217,6 +1217,16 @@ def register_bootstrap_options(cls, register):
             help="Overall timeout in seconds for each remote execution request from time of submission",
         )
 
+        register(
+            "--watch-filesystem",
+            type=bool,
+            default=True,
+            advanced=True,
+            help="Set to False if Pants should not watch the filesystem for changes. `pantsd` or `loop` "
+            "may not be enabled."
```

Comment:
It'd be good to explain why you might want to do this. Also, we tend to use `\n\n` to make the formatting nicer.

Maybe something like this?

```python
            help=(
                "Set to false to disable Pants watching the filesystem for changes.\n\n"
                "Pants normally watches the filesystem to make a best-effort attempt to "
                "restart builds invalidated by changes to relevant files made during a run. "
                "Disabling file watching increases the risk of invalid builds, but can be necessary, "
                 "e.g. when using Docker with Apple Silicon.\n\n"
                 "The global options `pantsd` and `loop` require file watching to be enabled."
             )
```

Reply:
I'll use that one verbatim. Thanks!